src/site/content/en/progressive-web-apps/install-criteria/index.md

```diff
@@ -68,7 +68,7 @@ Other browsers have similar criteria for installation, though there may be
 minor differences. Check the respective sites for full details:
 
 * [Edge](https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps#requirements)
-* [Firefox](https://developer.mozilla.org/Apps/Progressive/Add_to_home_screen#How_do_you_make_an_app_A2HS-ready)
+* [Firefox](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Installable_PWAs)
```

Comment:
```suggestion
* [Firefox](https://developer.mozilla.org/docs/Web/Progressive_web_apps/Installable_PWAs)
```

Reply:
by removing the locale it will load in whatever locale the user has.

Reply:
thanks